tutorials/notebooks/astropy-coordinates/1-Coordinates-Intro.ipynb

```diff
@@ -0,0 +1,634 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Astronomical Coordinates 1: Getting Started with astropy.coordinates\n",
+    "\n",
+    "## Authors\n",
+    "Adrian Price-Whelan\n",
+    "\n",
+    "## Learning Goals\n",
+    "* Create `astropy.coordinates.SkyCoord` objects using coordinate data and object names\n",
+    "* Use SkyCoord objects to become familiar with object oriented programming (OOP)\n",
+    "* Use a `SkyCoord` object to query the *Gaia* archive using `astroquery`\n",
+    "* Output coordinate data in different string representations\n",
+    "* Demonstrate working with 3D sky coordinates (including distance information for objects)\n",
+    "\n",
+    "## Keywords\n",
+    "coordinates, OOP, astroquery, gaia\n",
+    "\n",
+    "\n",
+    "## Summary\n",
+    "\n",
+    "Astronomers use a wide variety of coordinate systems and formats to represent sky coordinates of celestial objects. For example, you may have seen terms like \"right ascension\" and \"declination\" or \"galactic latitude and longitude,\" and you may have seen angular coordinate components represented as \"0h39m15.9s,\" \"00:39:15.9,\" or 9.81625º. `astropy.coordinates` provides tools for representing the coordinates of objects and transforming them between different systems. \n",
```

Comment:
```suggestion
    "Astronomers use a wide variety of coordinate systems and formats to represent sky coordinates of celestial objects. For example, you may have seen terms like \"right ascension\" and \"declination\" or \"galactic latitude and longitude,\" and you may have seen angular coordinate components represented as \"0h39m15.9s,\" \"00:39:15.9,\" or 9.81625º. Tools from `astropy.coordinates` provide  a representation the coordinates of objects and transforming them between different systems. \n",
```

Suggest rewriting so the sentence doesn't start with the package name.

Reply:
👍